include/maliput/routing/route.h

```diff
@@ -143,6 +144,20 @@ class Route final {
   /// @throws std::out_of_range When @p index is negative or >= `size()`.
   const Phase& Get(int index) const { return phases_.at(index); }
 
+  /// Indexes an api::LaneSRange in a Phase.
+  ///
+  /// @param phase_index The index of the Phase. It must be
+  /// non-negative and less than `size()`.
+  /// @param lane_s_range_index The index of the api::LaneSRange. It must be
+  /// non-negative and less than `Phase::lane_s_ranges().size()`.
+  /// @return The api::LaneSRange indexed at the @p phase_index -th
+  /// Phase at the @p lane_s_range_index -th position.
+  /// @throws std::out_of_range When any of the preconditions of
+  /// @p phase_index or @p lane_s_range_index are unmet.
+  const api::LaneSRange& GetLaneSRange(int phase_index, int lane_s_range_index) const {
+    return Get(phase_index).lane_s_ranges().at(lane_s_range_index);
+  }
```

Comment:
This will be used directly by the user. Can we maliput throw instead of a hard-to-track out of range? We should make a find before calling `at`
The Get method will need same treatment

Reply:
We can. But please note that we already have this in other parts of the API. Given that we are not try-catching anything I think it will be clear and easy to identify. Do you think we still need to do the range check with MALIPUT_THROW_UNLESS?

Reply:
I think that we can leave as it is and evaluate this once the API is started to be used.